### ISSUE TITLE
Move "occurrences" and "refresh" attribute reference documentation 

### ISSUE THREAD

**Issue by cwjohnston** [sensu/sensu-docs#120]
_From @calebhailey on October 19, 2017 19:53_

Documentation for "occurrences" and "refresh" is currently found on the Plugins reference documentation page, rather than the Checks documentation page. The reasoning behind this was because "occurrences" and "refresh" were historically not formally supported by Sensu itself, but rather, by the `sensu-plugin` gem. This [changed in version 0.26](https://sensuapp.org/docs/0.26/overview/changelog.html#core-v0-26-0) when we began shipping the built-in "occurrences" filter. 

We even explain the original reasoning [in the Plugin reference documentation](https://sensuapp.org/docs/latest/reference/plugins.html#sensu-plugin-definition-specification) as: 

> _NOTE: plugins based on the sensu-plugin gem derive configuration from [custom check definition attributes](https://sensuapp.org/docs/latest/reference/checks.html#custom-attributes). The configuration example(s) provided above, and the “specification” provided here are for clarification and convenience only (i.e. this “specification” is just an extension of the [check definition specification](https://sensuapp.org/docs/latest/reference/checks.html#check-definition-attributes), and not a definition of a distinct Sensu primitive)._

Basically, this note is no longer true, so we can remove it and move the reference docs to the check definition specification section. 

_Copied from original issue: sensu/sensu-docs#640_

**Comment by cwjohnston** [comment]
_From @majormoses on December 18, 2017 22:33_

Hm, agreed it needs to be updated the question is how to best convey the current state. I would say it is still not part of the core sensu and is delivered by an extension that is bundled with sensu. This also requires you to declare the filter to each handler so I do feel like in my mind it is still a custom attribute rather than is something natively in sensu that requires no additional changes to config to get it to work. Feel free to disagree or say I am bikeshedding on verbiage. Just my 2 cents.

**Comment by cwjohnston** [comment]
_From @majormoses on December 18, 2017 22:40_

To further clarify, what I mean to say is that it is not a native feature of sensu's framework but is a custom extension (even if it is bundled with core by default) therefore it is still a custom attribute. To me the distinction on a custom attribute is about whether it is part of the framework vs an extension (not as in running inside sensu runtime but filters, handlers, mutators, check scripts, metric scripts, etc) of it.

**Comment by cwjohnston** [comment]
_From @majormoses on December 18, 2017 22:43_

I don't think it should be removed as it still rings true other than it referring to `sensu-plugin` rather than `sensu-extensions-occurrences`.

**Comment by asachs01** [comment]
@sensu/success looks like the doc doesn't have the reference that Caleb initially pointed out. Is this issue still valid?

**Comment by cwjohnston** [comment]
@asachs01 I think https://docs.sensu.io/plugins/2.3/reference/#plugin-configuration is the current location of reference for occurrences and refresh attributes. 

My current thinking is that because we ship a filter extension with Sensu Core that honors these attributes, we should move them to the `event` reference with notes indicating that the `occurrences` filter must be explicitly configured to take action on these attributes.
